### DISCUSSION TITLE
Keyboard shortcuts

### DISCUSSION THREAD

**Discussion by fmarkos** [home-assistant/frontend#11289] [Other feature requests]
## The request
Keyboard shortcuts to switch between views (rooms in my case)

when keyboard or keypad number 1 is pressed, display "Living Room" view, 2 for view 2, etc.

## The alternatives
Using the mouse to select views

**Comment by iantrich** [comment]
This approach limits you to 10 views. I think adding it to the quick bar makes more sense: https://github.com/home-assistant/frontend/pull/7476 if done at all. Not convinced on this one.

**Reply by fmarkos** [reply]
Since I mainly use Home Assistant from a PC with a keyboard, keyboard shortcuts are most useful to me (I suspect this applies to many people) Switching Views (as in Rooms or tabs) with numbers or some other shortcut would be very useful.

**Comment by fmarkos** [comment]
I don't how to convice you how important keyboard shortcuts are for some people :-)
All I can say, is that I have made an html/js page that just listens for shortcuts and leads to the particular home assistant views.
![HA](https://user-images.githubusercontent.com/7732147/98932814-835f3700-24e8-11eb-94ae-d9fe6aa74d64.png)
`<script>
window.addEventListener('keydown', function (e) {
 switch(e.keyCode) {
  case 49:
    window.location.href ='https://********:8123/lovelace-admin/lab';
  break;
  case 97:
    window.location.href ='https://********:8123/lovelace-admin/lab';
  break;
  case 50: // 2
    window.location.href ='https://********:8123/lovelace-admin/west';
  break;
  case 98: // 2 keypad
    window.location.href ='https://********:8123/lovelace-admin/west';
  break;
  case 51: // 3
    window.location.href ='https://********:8123/lovelace-admin/bed-hall';
  break;
  case 99: // 3
    window.location.href ='https://********:8123/lovelace-admin/bed-hall';
  break;
  case 52: // 4
    window.location.href ='https://********:8123/lovelace-admin/computers';
  break;
  case 100: // 4
    window.location.href ='https://********:8123/lovelace-admin/computers';
  break;
  case 53: // 5
    window.location.href ='https://********:8123/lovelace-admin/cameras';
  break;
  case 101: // 5
    window.location.href ='https://********:8123/lovelace-admin/cameras';
  break;
  case 54: // 6
    window.location.href ='https://********:8123/lovelace-admin/automations';
  break;
  case 102: // 6
    window.location.href ='https://********:8123/lovelace-admin/automations';
  break;
  case 55: // 7
    window.location.href ='https://********:8123/lovelace-admin/basement';
  break;
  default:
  if(e.keyCode >= 65 && e.keyCode <= 80) {
        // console.log(e.keyCode);
      }
      break;
    }
  }
</script>
`
